proto/v1/common.proto

```diff
@@ -49,6 +49,8 @@ enum NoteKind {
   VULNERABILITY_ASSESSMENT = 11;
   // This represents an SBOM Reference.
   SBOM_REFERENCE = 12;
+  // This represents a secret.
+  SECRET = 18;
```

Comment:
Why not the next available value, i.e 13?

Reply:
good catch. will have another pr to fix

Reply:
Fixed in https://github.com/grafeas/grafeas/pull/629